docs/src/content/en/docs/getting-started/mcp-docs-server.mdx

```diff
@@ -40,34 +41,13 @@ Once it's installed in your IDE you can write prompts and assume the agent will
 
 **And more** - if you have a question, try asking your IDE and let it look it up for you.
 
-## Automatic Installation
```

Comment:
Do we not want to document this flag? Maybe it could be documented in the reference docs at least (if it's not already).

I do think it's useful to call out somehow that you can install it during a mastra installation.